### ISSUE TITLE
Unable to create fields in Production environemnt

### ISSUE THREAD

**Issue by mittalsunny22** [benedwards44/fieldcreator#8]
<!DOCTYPE html> <html> <head> <title>Salesforce.com Field Creator</title> <meta name="keywords" content="salesforce, salesforce.com, force.com, metadata, field creator, field create, fieldcreator, create fields" /> <meta name="description" content="Quickly create and deploy custom fields to your Salesforce.com environment" /> <meta name="author" content="Ben Edwards (ben@benedwards.co.nz)" /> <meta charset="UTF-8" /> <link rel="stylesheet" type="text/css" href="/static/bootstrap/css/bootstrap.min.5c7070ef655a.css" /> <link rel="stylesheet" type="text/css" href="/static/css/styles.db7cc5fbef83.css" /> <link rel="stylesheet" type="text/css" href="/static/css/bootstrap-switch.min.db48f5a585f0.css" />  <script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script> <script type="text/javascript" src="/static/js/main.8c4123838802.js"></script> <script src="/static/bootstrap/js/bootstrap.min.5869c96cc8f1.js"></script> <script type="text/javascript" src="/static/js/bootstrap-switch.min.09d61eed11b7.js"></script> <script> (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){ (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o), m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m) })(window,document,'script','//www.google-analytics.com/analytics.js','ga'); ga('create', 'UA-18288189-25', 'auto'); ga('send', 'pageview'); </script> </head> <body>  <nav class="navbar navbar-default"> <div class="container-fluid"> <div class="navbar-header"> <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button> </div> <div id="navbar" class="navbar-collapse collapse"> <ul class="nav navbar-nav navbar-left"> <li> <a href="/" style="padding:5px;"> <img src="/static/images/logo-small.png" alt="Field Creator" border="0" /> </a> </li> <li>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li> <li> <a href="https://cloudtoolkit.co" target="_blank">Salesforce Toolkit</a> </li> </ul> <ul class="nav navbar-nav navbar-right"> <li> <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top" style="padding-top:12px;"> <input type="hidden" name="cmd" value="_s-xclick" /> <input type="hidden" name="hosted_button_id" value="9J3KBL2Q4ZUS4" /> <input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donate_LG.gif" border="0" name="submit" title="PayPal - The safer, easier way to pay online!" alt="Donate with PayPal button" /> <img alt="" border="0" src="https://www.paypal.com/en_NZ/i/scr/pixel.gif" width="1" height="1" /> </form> </li> <li> <a href="https://github.com/benedwards44/fieldcreator" target="_blank">Source Code</a> </li> <li> <a href="https://twitter.com/benedwards44" target="_blank">@benedwards44</a> </li> <li> <a href="#"> <img src="/static/images/i-icon.png" data-toggle="modal" data-target="#infoModal" /> </a> </li> </ul> </div> </div> </nav> <div class="clear"></div> <div class="modal fade" id="infoModal" tabindex="-1" role="dialog" aria-labelledby="infoModalLabel" aria-hidden="true"> <div class="modal-dialog small_dialog"> <div class="modal-content"> <div class="modal-header"> <button type="button" class="close" data-dismiss="modal"> <span aria-hidden="true">&times;</span> <span class="sr-only">Close</span> </button> <h3 class="modal-title" id="apiModalLabel"> Salesforce Field Creator </h3> </div> <div class="modal-body"> <p> For any errors or issues, please feel free to contact me on twitter <a href="https://twitter.com/benedwards44" target="_blank">@benedwards44</a> or via email <a href="mailto:ben@benedwards.co.nz">ben@benedwards.co.nz</a>. </p> <p> I am constantly trying to update and improve this application, as well as other apps I'm working on. Feel free to contact me if you'd like to contribute or have any ideas. All my applications are open sourced on GitHub at <a href="http://github.com/benedwards44" target="_blank">http://github.com/benedwards44</a>. </p> <p> If you haven't already, check out my other apps at the Salesforce Toolkit Page:<br/> <a href="https://cloudtoolkit.co" target="_blank">https://cloudtoolkit.co</a> </p> </div> <div class="modal-footer"> <button type="button" class="btn btn-default" data-dismiss="modal">Close</button> </div> </div> </div> </div> <div class="col-md-12"> <div class="col-md-6 col-md-offset-3 text-center"> <h1>500 - Website Error</h1> <img alt="Doh" class="media-image" height="300" width="300" src="/static/images/oh-dear.png"> <div role="alert" class="alert alert-danger"> There appears to have been an error. The administrator has been notified. </div> </div> </div> </body> </html>

**Comment by esnorris** [comment]
Ran into the same issue creating fields in production (brand new org implementation).

**Comment by Allomo** [comment]
Seems to be a recurrent error, looks like a heroku server error, one day is working next is not.

**Comment by benedwards44** [comment]
I've ended up updating the app to Python 3 and Django 4, and upgraded the Heroku stack as well. This seems to have resolved this issue (for now...).
